Bit is removed in Qt6.6, fixes build with USE=sql ahead of next release.

USE=bluetooth is still broken, but is not essential (no revdeps), and
can instead be masked for the time being.
--- a/sip/QtCore/QtCoremod.sip
+++ b/sip/QtCore/QtCoremod.sip
@@ -24,3 +24,3 @@
 
-%Timeline {Qt_6_0_0 Qt_6_1_0 Qt_6_2_0 Qt_6_3_0 Qt_6_4_0 Qt_6_5_0}
+%Timeline {Qt_6_0_0 Qt_6_1_0 Qt_6_2_0 Qt_6_3_0 Qt_6_4_0 Qt_6_5_0 Qt_6_6_0}
 
--- a/sip/QtSql/qsqlresult.sip
+++ b/sip/QtSql/qsqlresult.sip
@@ -65,3 +65,5 @@
     int boundValueCount() const;
+%If (- Qt_6_6_0)
     QList<QVariant> &boundValues() const;
+%End
     QString executedQuery() const;
